docs/blog/newsletter-december-2021.rst

```diff
@@ -0,0 +1,121 @@
+
+One more tip: our `June 2021 newsletter <https://www.writethedocs.org/blog/newsletter-june-2021/>`_ included a relevant item about balancing writing new docs with ongoing docs maintenance, which may also be helpful.
+
+-----------------------------------------
+Answers to “Why are good docs important?”
+-----------------------------------------
+
+What is the importance of good documentation? This question can come up a lot - during an interview, project planning, or just a chat with a colleague. Do you have a good answer?  
+
+You’ve got plenty of options. For example, that good documentation eases the strain on the support staff or minimizes friction for customers, enabling them to complete tasks quickly. Or even more ideas: that good documentation acts as a sales tool, helping with both advertising and lead generation; it shows product commitment, and helps close profitable deals with potential customers. It's also valuable internally as a source of truth, allowing everyone in the organization to work smarter rather than harder.
+
+For more on this, Lari Maza discusses good documentation in her article `“How to Write Good Documentation” <https://medium.com/larimaza-en/how-to-write-good-documentation-e19c70dc67f0>`__. According to Maza, writers should anticipate customer questions, keep within scope, and expand information with links and images.
+
+----------------------------------
+Standards in documentation tooling
+----------------------------------
+
+We had a great discussion this month about the fragmentation of docs-as-code tooling, sparked by `Fabrizio Ferri-Benedetti's article <https://passo.uno/docs-as-code-tools-open-standards/>`__. The article talks about the often-fragile docs-as-code toolchains that companies run for themselves. When tools are hard to maintain or don't work well, they hinder growth, speed, and collaboration. He argues that these problems are caused by systemic issues: we’re missing a standard markup language, universal documentation rendering, and a version control system designed for docs.
+
+Documentation rendering was contentious, but folks saw more promise in the markup idea. DITA is heavyweight; Markdown lets you get on with writing, but is neither standardised nor designed for docs; and the alternatives aren't widely used. Some community members defended AsciiDoc as meeting all the criteria - it's interesting that it's currently going through `a standardization process via Eclipse <https://projects.eclipse.org/projects/asciidoc.asciidoc-lang>`__.
+
+The article's problems with Git - in particular, that it's hard to learn - aren't just a problem for writers, but are complained about by many developers too. There is a counter-argument to needing a new system: instead, we should take the tools we have, but make them work better for use cases. Perhaps we don't need a new version control system, but better Git clients designed for docs workflows…
```

Comment:
> The article's problems with Git - in particular, that it's hard to learn - aren't just a problem for writers, but are complained about by many developers too. 

This reads kinda awkward, but I don't have a great rewording. How about

Many writers and developers complain about git, most often that it is hard to learn.

Reply:
How about: The article's main problem with Git, often expressed by writers as well as developers, is that it is hard to learn.

Reply:
Better for sure

Reply:
Much better, thanks!